.pre-commit-config.yaml

```diff
@@ -49,3 +49,7 @@ repos:
             testing/.*.json|
             testing/.*.xml
         )$
+ci:
+  autoupdate_schedule: monthly
+  autofix_prs: false
+  autoupdate_branch: 'dependency-updates'
```

Comment:
If you like you can generate a custom message here. It defaults to: 

`autoupdate_commit_msg: '[pre-commit.ci] pre-commit autoupdate'`

Ref: https://pre-commit.ci/#configuration

Reply:
ok, I've added `'build(pre-commit): pre-commit autoupdate by ci'` to keep it in the same style